### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Screen Reader: Narrator is not reading the label names for all the controls in the page

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#170]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ScreenReader; #Narrator; #Chrome; #NVDA; #MAS1.3.1

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS 1.3.1

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authentication and press enter
6. Verify whether narrator is reading label names for all the controls or not

**Actual Result:**
Narrator is not reading the label names for all the controls in the page

**Expected Result:**
Narrator should read the label names for all the controls in the page

**User Impact:**
Screen Reader users finds difficulty while working with the application if narrator does not read the label names 

**Note:**
Similar issue is also found in 
Web Server- Authorization
Web Server- Authorization- Create Rule
Web Server- IP Restrictions
Web Server- Url Rewrite- Outbound Rules

**Attachments:**
[Bug170_web server_authentication_narrator not reading label names.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331864/Bug170_web.server_authentication_narrator.not.reading.label.names.pptx)

![bug170_web server_authorization_narrator not reading label names](https://user-images.githubusercontent.com/42638450/44830424-411ae980-ac3f-11e8-895b-8ec2aad83a6f.png)
![bug170_web server_ip restrictions_narrator not reading label names1](https://user-images.githubusercontent.com/42638450/44830425-41b38000-ac3f-11e8-99d4-73eb92f347bb.png)
![bug170_web server_ip restrictions_narrator not reading label names2](https://user-images.githubusercontent.com/42638450/44830426-41b38000-ac3f-11e8-9cda-063f26dafee7.png)
![bug170_web server_outbound rules_narrator not reading label names1](https://user-images.githubusercontent.com/42638450/44830427-41b38000-ac3f-11e8-89b4-b905a004a770.png)
![bug170_web server_outbound rules_narrator not reading label names2](https://user-images.githubusercontent.com/42638450/44830428-424c1680-ac3f-11e8-8dec-32cd7150740e.png)
![bug170_web server_outbound rules_narrator not reading label names3](https://user-images.githubusercontent.com/42638450/44830429-424c1680-ac3f-11e8-81f4-789213b48195.png)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
